Ports/mc/patches/0001-config-config.sub-Add-SerenityOS-as-portable-system.patch

```diff
@@ -1,29 +0,0 @@
-From 34a4ba3c619add44391b7d7a7422babab9993fbb Mon Sep 17 00:00:00 2001
-From: Kenneth Myhra <kennethmyhra@gmail.com>
-Date: Sat, 25 Dec 2021 19:01:58 +0100
-Subject: [PATCH 1/3] config/config.sub: Add SerenityOS as portable system
-
-- [ ] Local?
-- [X] Should be merged to upstream?
-- [ ] Resolves issue(s) with our side of things
-- [ ] Hack
----
- config/config.sub | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/config/config.sub b/config/config.sub
-index 1d8e98b..85b7fde 100755
---- a/config/config.sub
-+++ b/config/config.sub
-@@ -1363,7 +1363,7 @@ case $os in
- 	# The portable systems comes first.
- 	# Each alternative MUST end in a * to match a version number.
- 	# -sysv* is not here because it comes later, after sysvr4.
--	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
-+	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* | -serenity* \
- 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
- 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
- 	      | -sym* | -kopensolaris* | -plan9* \
--- 
-2.25.1
-
```

Comment:
Wouldn't it be better to delete this file in the next commit where you update the version number and download the fresh config.sub

Reply:
No, because I already replaced it by `use`_fresh_config_sub` in this commit.

Reply:
Dammit, I'm blind!:)